.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -296,6 +297,22 @@ Minikube is the only caveat - it's not reachable publicly so don't run a proxy t
       * [ ] Restart the agent after token TTL expires to see if it reuses the same identity.
     * [ ] Force cluster CA rotation
 
+### Kubernetes Pod RBAC
+
+* [ ] Verify the following scenarios for `kubernetes_resources`:
```

Comment:
Should we add a specific case for filtering listed pods? When Kubernetes role can see more pods than Teleport role?